osu.Game/Overlays/Comments/CommentEditor.cs

```diff
@@ -167,33 +170,59 @@ private void load(OverlayColourProvider colourProvider)
         protected override void LoadComplete()
         {
             base.LoadComplete();
-            Current.BindValueChanged(_ => updateCommitButtonState(), true);
-            apiState.BindValueChanged(updateStateForLoggedIn, true);
+            Current.BindValueChanged(_ => updateState());
```

Comment:
Any reason this one isn't `AddOnce`'d?

Reply:
`Current` is the comment text in this context, and `updateState()` is now responsible for disabling the commit button, so I want this particular interaction to be instant.